index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+      <div class="nav">
+        <header>
+            <div><a href="#" class="logo"><h1>Welcome</h1></a></div>
+        </header>
+        <nav>
+                
+            <div class="Togglebutton"><a href="#"><img class="MenuIcons" src="images/icons/icon-Menu.png" alt="togglebutton"></a></div>
+        </nav>
+      </div> 
```

Comment:
**Optional**

- [ ] As suggested by the last reviewer. It will be nice if we follow the project requirement. This enables us to use Semantic HTML for best practices. 

```
<header>
  <a href="#"> -- logo goes here -- </a>
  <nav>
    <button> -- menu icon goes here -- </button>
  </nav>
</header>
```
The `header tag` should wrap the content, not the current `div tag`